app/src/components/pages/AboutUs.tsx

```diff
@@ -13,28 +13,97 @@ class AboutUs extends Component {
-					lokalen Einzelhandel sowie lokalen Unternehmer:innen zu helfen.
+						Wir sind ein Team aus 8 Entwickler*innen, das sich über den
+						<a target="_blank" href="https://wirvsvirushackathon.org/">
+							<b> #WirVsVirus</b>-Hackathon{' '}
+						</a>
+						der Bundesregierung kennengelernt haben. Uns vereint das gemeinsame Ziel,
+						dem lokalen Einzelhandel sowie lokalen Unternehmer:innen zu helfen.
+					</p>
+					<p className="about--description-text">
+						{' '}
+						Viel Spaß mit der Website wünschen Euch{' '}
 					</p>
-					<p className="about--description-text">	Viel Spaß mit der Website wünschen Euch </p>
 					<div className="about--description-text about--people-grid">
-						<div>👨‍💻
-							<a className="about--people-link" href="https://github.com/redalertexpert">Felix</a>,</div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/Koenneker">Ferdinand</a>,</div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/hf2000510">Henri</a>,</div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/L04L3R">Jens</a>,</div>
-							<div>👩‍💻 <a className="about--people-link" href="https://github.com/Juliaetta">Julia</a>,</div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/JulianLueders">Julian</a>,</div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/TheWoozyDude">Stefan</a> und </div>
-							<div>👨‍💻 <a className="about--people-link" href="https://github.com/tobiaswaelde">Tobias</a></div>
+						<div>
+							👨‍💻
+							<a
+								className="about--people-link"
+								href="https://github.com/redalertexpert"
+							>
+								Felix
+							</a>
+							,
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a className="about--people-link" href="https://github.com/Koenneker">
+								Ferdinand
+							</a>
+							,
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a className="about--people-link" href="https://github.com/hf2000510">
+								Henri
+							</a>
+							,
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a className="about--people-link" href="https://github.com/L04L3R">
+								Jens
+							</a>
+							,
+						</div>
+						<div>
+							👩‍💻{' '}
+							<a className="about--people-link" href="https://github.com/Juliaetta">
+								Julia
+							</a>
+							,
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a
+								className="about--people-link"
+								href="https://github.com/JulianLueders"
+							>
+								Julian
+							</a>
+							,
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a
+								className="about--people-link"
+								href="https://github.com/TheWoozyDude"
+							>
+								Stefan
+							</a>{' '}
+							und{' '}
+						</div>
+						<div>
+							👨‍💻{' '}
+							<a
+								className="about--people-link"
+								href="https://github.com/tobiaswaelde"
+							>
+								Tobias
+							</a>
+						</div>
```

Comment:
Ist das nur Reformat?

Reply:
Probably yes. I did not change anything on this part of code. I guess it was my "Prettier" extension because the lines were >80 chars long. I will fix that with an extra component in the next commit.